proto/shentu/oracle/v1alpha1/tx.proto

```diff
@@ -109,3 +113,44 @@ message MsgDeleteTask {
 }
 
 message MsgDeleteTaskResponse {}
+
+message MsgCreateTxTask {
+    option (gogoproto.equal) = false;
+    option (gogoproto.goproto_getters) = false;
+
+    string creator = 1 [ (gogoproto.moretags) = "yaml:\"creator\"" ];
+    string chain_id = 2 [ (gogoproto.moretags) = "yaml:\"chain_id\"" ];
+    // the bytes of business chain transaction that is going to be evaluated
+    bytes tx_bytes = 3 [ (gogoproto.moretags) = "yaml:\"tx_bytes\"" ];
+    repeated cosmos.base.v1beta1.Coin bounty = 4 [ (gogoproto.moretags) = "yaml:\"bounty\"", (gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins" ];
+    google.protobuf.Timestamp valid_time = 5 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"valid_time\""];
+}
+
+message MsgCreateTxTaskResponse {
+    option (gogoproto.equal) = false;
+    option (gogoproto.goproto_getters) = false;
+    // sha256 hash of the business chain transaction
+    bytes tx_hash = 1;
+}
+
+message MsgTxTaskResponse {
+    option (gogoproto.equal) = false;
+    option (gogoproto.goproto_getters) = false;
+
+    bytes tx_hash = 1 [ (gogoproto.moretags) = "yaml:\"tx_hash\"" ];
+    int64 score = 2 [ (gogoproto.moretags) = "yaml:\"score\"" ];
+    string operator = 3 [ (gogoproto.moretags) = "yaml:\"operator\"" ];
+}
+
+message MsgTxTaskResponseResponse {}
+
+message MsgDeleteTxTask {
+    option (gogoproto.equal) = false;
+    option (gogoproto.goproto_getters) = false;
+
+    bytes tx_hash = 1 [ (gogoproto.moretags) = "yaml:\"tx_hash\"" ];
+    bool force = 2 [ (gogoproto.moretags) = "yaml:\"force\"" ];
+    string deleter = 3 [ (gogoproto.moretags) = "yaml:\"deleter\"" ];
```

Comment:
Just borrow the field name from existing structure. 
But I'm with you on naming this field `from`.

Reply:
I think we should refactor. the old/existing oracle code is really outdated.

Reply:
Let's refactor the existing structure.

Reply:
I've done a bit check on Shentu mainnet. It seems the `MsgDeleteTask` has never been called. 
So if it would not affect back compatibility, I'll do the refactoring for the existing `MsgDeleteTask`.

Reply:
done with latest commit.